### PR TITLE
Debug hlt - L2MuonProducer: adding EnableGEMMeasurement and GEMRecSegmentLabel parameters

### PR DESCRIPTION
**Pull request by jshlee**
L2MuonProducer: adding EnableGEMMeasurement and GEMRecSegmentLabel parameters to enable HLT. EnableGEMMeasurement is set to false by default

### CODE REVIEW

**Comment by dildick** [comment]
Shouldn't we enable this in gemCustoms in the HLT customization step?

**Comment by jshlee** [comment]
Yes, it should be enabled with gemCustoms. But since this bit is for L2, I don't think it will work anyways.

**Comment by dildick** [comment]
Well, I'd rather play it safe and set all flags to True in gemCustoms as the release managers asked us to do. It's also a less error-prone solution.

**Comment by jshlee** [comment]
Yes, that should be fine. I didn't change gemCustoms. I only added this for the default non gem HLT chain.

File: RecoMuon/L2MuonProducer/python/L2Muons_cfi.py [hunk 1]
```diff
@@ -36,7 +36,8 @@
             EnableDTMeasurement = cms.bool(True),
             RPCRecSegmentLabel = cms.InputTag("rpcRecHits"),
             Propagator = cms.string('SteppingHelixPropagatorL2Any'),
-            EnableCSCMeasurement = cms.bool(True)
+            EnableCSCMeasurement = cms.bool(True),
+            EnableGEMMeasurement = cms.bool(True)
         ),
```
Comment (dildick) [inline RecoMuon/L2MuonProducer/python/L2Muons_cfi.py:41]: I would set this to false by default

File: RecoMuon/L2MuonProducer/python/L2Muons_cfi.py [hunk 2]
```diff
@@ -64,7 +65,9 @@
             EnableDTMeasurement = cms.bool(True),
             RPCRecSegmentLabel = cms.InputTag("rpcRecHits"),
             Propagator = cms.string('SteppingHelixPropagatorL2Any'),
-            EnableCSCMeasurement = cms.bool(True)
+            EnableCSCMeasurement = cms.bool(True),
+            EnableGEMMeasurement = cms.bool(True)
+
```
Comment (dildick) [inline RecoMuon/L2MuonProducer/python/L2Muons_cfi.py:70]: I would set this to false by default

**Comment by jshlee** [comment]
yes, your right.
forgot to change that bit since it doesn't seem to be used in the L2...

**Pull request merged**